spec.bs

```diff
@@ -497,6 +506,22 @@ The specification adds the following steps to the [=HTTP fetch=] algorithm, befo
 1. If |redeem response result| is a [=network error=], return |issue response result|.
 
 
+Integration with XMLHttpRequest {#xhr}
+====================================
+
+An XMLHttpRequest has an associated {{PrivateToken}} object specifying the {{OperationType}} to execute against the request.
+
+The <dfn><code>setPrivateToken()</code></dfn> method accepts a {{PrivateToken}} |privateToken| and must do the following:
+<ol>
+ <li><p>If [=state=] is not "opened", then throw an
+ "{{InvalidStateError!!exception}}" {{DOMException}}.
+ <li> If <a>this</a>'s
+    <a href="https://xhr.spec.whatwg.org/#send-flag">`send()` flag</a> is set,
+    then [=throw=] an "{{InvalidStateError!!exception}}" {{DOMException}}.
+ <li> Run [=set private token properties for request from private token=] on |privateToken|
```

Comment:
nit: Add periods to this step and the next one.

Reply:
done